server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -572,6 +572,12 @@ public class ShuffleServerConf extends RssBaseConf {
           .withDescription(
               "keep alive time of thread pool that used for calc summary metric, in SECONDS.");
 
+  public static final ConfigOption<Boolean> APP_LEVEL_SHUFFLE_BLOCK_SIZE_METRIC_ENABLED =
+      ConfigOptions.key("rss.server.block.size.metric.enabled")
+          .booleanType()
+          .defaultValue(true)
```

Comment:
I think app-level metrics should not be enabled by default as this might overwhelm metric servers on large Spark clusters, especially if not used.